apps/lockfile-explorer-server/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import express from 'express';
+import cors from 'cors';
+import yaml from 'js-yaml';
+const app: express.Application = express();
+import fs from 'fs';
+import path from 'path';
+const port: number = 8091;
+
+app.use(cors());
```

Comment:
Instead of using CORS, it'd be cool to just serve the website with the same server.

The `lockfile-explorer-server` project could get renamed to `lockfile-explorer` (and what's currently called `lockfile-explorer` could get renamed to `lockfile-explorer-webapp`), this project could take a dependency on the webapp project, and it could copy the `dist` folder into its build output (under like `lib/assets`) and serve that statically.

Reply:
We should think a bit about how this project will get published to NPM.  

Maybe the "main" project is the Node.js service + accompanying CLI tool, whereas the web app is just a `.js` asset that gets copied into the `dist` folder.  In this way the published package would actually be the service, and the web app would be a devDependency.

Reply:
>Maybe the "main" project is the Node.js service + accompanying CLI tool, whereas the web app is just a .js asset that gets copied into the dist folder. In this way the published package would actually be the service, and the web app would be a devDependency.

Yeah that's basically what I was suggesting.

Reply:
I've implemented this idea in a5437f8c1a37065548130a81fd30ae1e114900f8